.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
## Your Readme
![image](https://user-images.githubusercontent.com/93039359/224509292-024d3085-4a72-4bde-8156-e697823fbff5.png)
## Descriptive Readme
![image](https://user-images.githubusercontent.com/93039359/224509379-e98a63b6-07fd-49a2-93b8-2f943c1985e5.png)

- [ ] It would be nice if you make your Tech Stack Heading look like the below ⬇️ to make your readme clean and professional 😊